lib/cli/interactive-setup/index.js

```diff
@@ -12,15 +12,20 @@ const steps = {
 };
 
 module.exports = async (context) => {
-  context = { ...context, inquirer, history: new Map() };
+  context = { ...context, inquirer, history: new Map(), stepDurations: new Map() };
```

Comment:
I think `durations` do not really belong to context (which is about input data for steps processing).

I would keep them aside of `context`, and from this module I'd just return `{ history, durations }`, not context as a whole (`context` is input argument, so returning it doesn't seem justified)